Omise/Models/AuthType.cs

```diff
@@ -0,0 +1,14 @@
+using System.Runtime.Serialization;
+
+namespace Omise.Models
+{
+    public enum AuthType
+    {
+        [EnumMember(Value = null)]
+         None,
```

Comment:
Does it need `None` enum member to handle `null`? Can it handle from property declaration?

Reply:
I followed the `FlowTypes.cs` enum structure

Reply:
Okay, can you change to name to plural name?

Reply:
changed @nuxzero